R/rd-data.R

```diff
@@ -160,6 +160,28 @@ as_data.tag_examples <- function(x, ...,
   paste(html, collapse = "")
 }
 
+make_print_handler <- function(options = NULL, inline = NULL) {
+  evaluate::new_output_handler(
+    value = function(x) knitr::knit_print(x, options, inline)
+  )
+}
+
+example_chunk_opts <- function(topic, obj_id) {
+  # these are the default pkgdown figure settings
+  # in list format, with an additional fig.path specified.
+  list(
+    dev = "png",
+    dpi = 96,
+    fig.retina = 2,
+    dev.args = list(),
+    fig.width = 7.291667,
+    fig.height = 4.506593,
+    fig.asp = 0.618047,
+    fig.path = paste0("figures/", topic, "-", obj_id(topic))
```

Comment:
Need https://github.com/thomasp85/gganimate/pull/255 for `fig.path` to have an effect.